File: inrt/src/main/java/com/stardust/auojs/inrt/launch/GlobalProjectLauncher.java
package com.stardust.auojs.inrt.launch;

import android.app.Activity;
import android.content.Context;

import com.stardust.app.GlobalAppContext;
import com.stardust.auojs.inrt.App;

/**
 * Created by Stardust on 2018/3/21.
 */

public class GlobalProjectLauncher extends AssetsProjectLauncher {

    private static GlobalProjectLauncher sInstance;

    public static GlobalProjectLauncher getInstance() {
        if (sInstance == null)
            sInstance = new GlobalProjectLauncher(GlobalAppContext.get());
        return sInstance;
    }

    GlobalProjectLauncher(Context context) {
        super("project", context);
    }
}

File: inrt/src/main/java/com/stardust/auojs/inrt/launch/GlobalProjectLauncher.kt
package com.stardust.auojs.inrt.launch

import android.annotation.SuppressLint
import com.stardust.app.GlobalAppContext

/**
 * Created by Stardust on 2018/3/21.
 */

@SuppressLint("StaticFieldLeak")
object GlobalProjectLauncher: AssetsProjectLauncher("project", GlobalAppContext.get())
